variables.tf

```diff
@@ -254,3 +254,8 @@ variable "enabled" {
   description = "Whether the database resources should be created"
 }
 
+variable "enabled_cloudwatch_logs_exports" {
+  type        = list(string)
+  default     = []
+  description = "Which log types to export to CloudWatch Logs"
```

Comment:
How about using the description from https://www.terraform.io/docs/providers/aws/r/rds_cluster.html#enabled_cloudwatch_logs_exports ? In particular I was wondering what the valid values were.

Reply:
Good point.  Updated.